pkg/server/metrics/metrics.go

```diff
@@ -34,8 +34,8 @@ const (
 )
 
 var (
-	// Use buckets ranging from 5 ms to 12.5 seconds.
-	latencyBuckets = []float64{0.005, 0.025, 0.1, 0.5, 2.5, 12.5}
+	// Use buckets ranging from 10 ns to 12.5 seconds.
```

Comment:
nit: 10e-7 is 100 ns

10 ns sounds too small, that's just tens of CPU cycles, even for a local write I doubt 10 ns is enough, so let's just update the comment to 100ns.

Reply:
Fair. I kept masking it smaller until the units actually didn't show up to confirm ```le``` was working.